packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -480,19 +480,11 @@ class FlutterDriver {
 
     // If the item is already visible then we're done.
     bool isVisible = false;
-    try {
```

Comment:
Does the new version of the code really work when the item isn't initially visible? The DriverError test here used to be needed, what changed?